rocAL/rocAL_pybind/examples/READERS_TEST_FILE.sh

```diff
@@ -168,7 +168,7 @@ if [[ rocAL_api_caffe_reader -eq 1 ]]; then
 
     # Mention dataset_path
     # Classification
-    data_dir=$ROCAL_DATA_PATH/caffe/classification/ilsvrc12_train_lmdb/
+    data_dir=$ROCAL_DATA_PATH/caffe/classification/
```

Comment:
@swetha097 Have these paths been updated in the mivisionx-data repo as well?

Reply:
Yes Lakshmi, that has been updated